src/flag_gems/__init__.py

```diff
@@ -1,180 +1,207 @@
-    lib.impl("triu", triu, "CUDA")
-    lib.impl("topk", topk, "CUDA")
-    lib.impl("var_mean.correction", var_mean, "CUDA")
-    lib.impl("linalg_vector_norm", vector_norm, "CUDA")
-    lib.impl("where.self_out", where_self_out, "CUDA")
-    lib.impl("where.self", where_self, "CUDA")
-    lib.impl("where.ScalarSelf", where_scalar_self, "CUDA")
-    lib.impl("where.ScalarOther", where_scalar_other, "CUDA")
-    lib.impl("max", max, "CUDA")
-    lib.impl("max.dim", max_dim, "CUDA")
-    lib.impl("min", min, "CUDA")
-    lib.impl("min.dim", min_dim, "CUDA")
-    lib.impl("amax", amax, "CUDA")
-    lib.impl("argmax", argmax, "CUDA")
-    lib.impl("prod", prod, "CUDA")
-    lib.impl("prod.dim_int", prod_dim, "CUDA")
-    lib.impl("sum", sum, "CUDA")
-    lib.impl("sum.dim_IntList", sum_dim, "CUDA")
-    lib.impl("all", all, "CUDA")
-    lib.impl("all.dim", all_dim, "CUDA")
-    lib.impl("all.dims", all_dims, "CUDA")
-    lib.impl("any", any, "CUDA")
-    lib.impl("any.dim", any_dim, "CUDA")
-    lib.impl("any.dims", any_dims, "CUDA")
-    lib.impl("log_softmax.int", log_softmax, "AutogradCUDA")
-    lib.impl("outer", outer, "AutogradCUDA")
-    lib.impl("cross_entropy_loss", cross_entropy_loss, "AutogradCUDA")
-    lib.impl("scatter.src", scatter, "CUDA")
-    lib.impl("scatter.reduce", scatter, "CUDA")
-    lib.impl("gather", gather, "CUDA")
-    lib.impl("isclose", isclose, "CUDA")
-    lib.impl("allclose", allclose, "CUDA")
-    lib.impl("fill.Scalar", fill_scalar, "CUDA")
-    lib.impl("fill.Tensor", fill_tensor, "CUDA")
-    lib.impl("flip", flip, "CUDA")
-    lib.impl("slice_scatter", slice_scatter_v2, "CUDA")
-    lib.impl("select_scatter", select_scatter, "CUDA")
-    lib.impl("index_select", index_select, "CUDA")
-    lib.impl("tile", tile, "CUDA")
-    lib.impl("masked_fill.Tensor", masked_fill, "CUDA")
-    lib.impl("masked_fill.Scalar", masked_fill, "CUDA")
-    lib.impl("masked_fill_.Tensor", masked_fill_, "CUDA")
-    lib.impl("masked_fill_.Scalar", masked_fill_, "CUDA")
-    lib.impl("_unique2", _unique2, "CUDA")
-    lib.impl("_upsample_bicubic2d_aa", _upsample_bicubic2d_aa, "CUDA")
-    lib.impl("upsample_nearest2d", upsample_nearest2d, "CUDA")
-    lib.impl("nonzero", nonzero, "CUDA")
-    lib.impl("repeat", repeat, "CUDA")
-    lib.impl("masked_select", masked_select, "CUDA")
-    lib.impl("stack", stack, "CUDA")
-    lib.impl("hstack", hstack, "CUDA")
-    lib.impl("cat", cat, "CUDA")
-    lib.impl("repeat_interleave.self_int", repeat_interleave_self_int, "CUDA")
-    lib.impl("vstack", vstack, "CUDA")
-    lib.impl("repeat_interleave.Tensor", repeat_interleave_tensor, "CUDA")
-    lib.impl("repeat_interleave.self_Tensor", repeat_interleave_self_tensor, "CUDA")
-    lib.impl("randperm", randperm, "CUDA")
-    lib.impl("diag", diag, "CUDA")
+def enable(lib=aten_lib, unused=None):
+    if unused is None:
+        unused = []
+    Register(
+        (
+            ("abs", abs, Autograd.disable),
```

Comment:
I prefer on/off rather than enable/disable, cause on/off represent for status while enable/disable are procedure. But it's not that important.

Reply:
On or off does not show a lack of support, and user feel that they can be both on and off